Add the DDL files(ejbtimer) for PostgreSQL(eclipse-ee4j#22850)

endtak opened the pull request:
Add PostgreSQL as the DB to use for the EJB timer service.
This modification adds the following ddl files.

*  glassfish/lib/install/databases/ejbtimer_postgres.sql
*  glassfish/lib/install/databases/upgrade/ejbtimer_upgrade_postgres.sql

Signed-off-by: endtak <endo.takafumi@fujitsu.com>

Commit:
Add the DDL files(ejbtimer) for PostgreSQL(eclipse-ee4j#22850)

Signed-off-by: endtak <endo.takafumi@fujitsu.com>

smillidge reviewed (commented):
Why have you made this copyright Oracle and why 2018?

endtak committed:
Fix for copyright.

Signed-off-by: endtak <endo.takafumi@fujitsu.com>

endtak commented:
sry, I was not very conscious of copyright. Commit the modifications.


The email address has been changed according to our company policy. As a result, I changed my Eclipse Foundation account email address and reran ECA with the new email address.
This pull request is checked against the old email address, so it seems to fail with the new email address. I changed the signature of the first commit (eeeb474) to a new email address, but I couldn't solve the problem.
If you issue a new pull request, there may be a problem with the system because the ECA check is passed.

endtak commented:
Since ECA check does not pass, make a pull request again.
